src/test/java/duke/taskListTest/TaskListTest.java

```diff
@@ -0,0 +1,44 @@
+package duke.taskListTest;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+
+import duke.parser.DateTimeParser;
+import duke.tasklist.TaskList;
+import duke.tasks.Deadline;
+import duke.tasks.Event;
+import duke.tasks.Task;
+import duke.tasks.ToDo;
+
+
+public class TaskListTest {
+    private TaskList testList = new TaskList();
+
+    public void loadTestList(TaskList testList) {
+        testList.add(new ToDo("clean room"));
+        testList.add(new Deadline("do homework", DateTimeParser.deadlineDateParse("2021-12-31")));
+        testList.add(new Event("university fair", DateTimeParser.eventDateTimeParse("2021-11-30 0101")));
+    }
+
+
+    @Test
+    public void getTaskTest() {
+        loadTestList(testList);
+        int index = 1;
+        assertEquals(new ToDo("clean room").toString(), testList.get(index - 1).toString());
+    }
```

Comment:
I feel that this is not tested in isolation since bugs for ToDo/Deadline/Event can affect this test case.